MashUp-iOS/MashUp-iOS/Source/Screen/Seminar Schedule/Seminar Detail/SeminarDetailViewController.swift

```diff
@@ -0,0 +1,194 @@
+            .disposed(by: self.disposeBag)
+        
+        self.collectionView.rx.itemSelected.map { .didSelectSeminar(at: $0.item) }
+            .bind(to: reactor.action)
+            .disposed(by: self.disposeBag)
+    }
+    
+    private func render(_ reactor: Reactor) {
+        //        reactor.state.map { $0.sections }
+        //        .distinctUntilChanged()
+        //        .withUnretained(self)
+        //        .onMain()
+        //        .subscribe(onNext: { $0.applySections($1) })
+        //        .disposed(by: self.disposeBag)
+    }
+    
+    private func consume(_ reactor: Reactor) {
+        //        reactor.pulse(\.$step).compactMap { $0 }
+        //        .withUnretained(self)
+        //        .onMain()
+        //        .subscribe(onNext: { $0.move(to: $1) })
+        //        .disposed(by: self.disposeBag)
+    }
+    
+    private func applySections(_ sections: [Section]) {
+        var snapshot = Snapshot()
+        snapshot.appendSections(sections)
+        sections.forEach { snapshot.appendItems($0.items, toSection: $0) }
+        self.dataSource.apply(snapshot, animatingDifferences: true)
+    }
+    
+    private lazy var dataSource = self.dataSourceOf(self.collectionView)
+    
+    private let collectionView = UICollectionView(
+        frame: .zero,
+        collectionViewLayout: UICollectionViewFlowLayout()
+    )
+    
+}
+// MARK: - Setup
+extension SeminarDetailViewController {
+    
+    private func setupUI() {
+        self.setupAttribute()
+        self.setupLayout()
+    }
+    
+    private func setupAttribute() {
+        self.view.backgroundColor = .white
+        self.collectionView.do {
+            $0.backgroundColor = .clear
+            $0.registerCell(SeminarDetailCell.self)
+            $0.registerSupplementaryView(SeminarDetailHeaderView.self)
+            $0.registerSupplementaryFooterView(SeminarDetailFooterView.self)
+            $0.collectionViewLayout = self.collectionViewLayout()
+        }
+        self.goToBackButton.do {
+            $0.titleLabel?.text = "돌아가기"
+            $0.titleLabel?.font = .pretendardFont(weight: .medium, size: 14)
+            $0.setBackgroundColor(.primary100, for: .normal)
+            $0.setTitleColor(.primary500, for: .normal)
+        }
+        self.bottomSafeAreaView.do {
+            $0.backgroundColor = .white
+        }
+    }
+    
+    private func setupLayout() {
+        self.view.addSubview(self.collectionView)
+        self.view.addSubview(self.goToBackButton)
+        self.collectionView.snp.makeConstraints {
+            $0.top.equalToSuperview().inset(24)
+            $0.leading.trailing.bottom.equalToSuperview()
+        }
+        self.view.addSubview(self.bottomSafeAreaView)
+        self.goToBackButton.snp.makeConstraints {
+            $0.height.equalTo(52)
+            $0.bottom.equalToSuperview().inset(28)
+            $0.trailing.equalToSuperview().inset(20)
+            $0.leading.equalToSuperview().inset(20)
```

Comment:
```suggestion
            $0.trailing.leading.equalToSuperview().inset(20)
```
이렇게 축약해서도 표기가 가능합니답 참고차 알아두시면 좋을 것 같아서 공유드립니답 🙇‍♂️